src/Microsoft.Identity.Client/User.cs

```diff
@@ -47,37 +46,34 @@ internal User(User other)
             IdentityProvider = other.IdentityProvider;
         }
 
+        public User(string identifier, string displayableId, string name, string identityProvider)
```

Comment:
We must not allow creation of user object publicially. This should be an internal detail